exchanges/poloniex/poloniex_test.go

```diff
@@ -2263,7 +2256,7 @@ func TestIntervalString(t *testing.T) {
 	var err error
 	var is string
 	for key, val := range params {
-		is, err = IntervalString(key)
+		is, err = IntervalToString(key)
 		require.Equal(t, val.IntervalString, is)
```

Comment:
🚧 Change request:
* `is` is weird. Please don't 🙂 
* Not sure why we have vars outside their usage scope. Can you scope them into it?
```suggestion
	for key, val := range params {
		s, err := IntervalToString(key)
		require.Equal(t, val.IntervalString, s)
```